Source/ViewModel/MLCardFormViewModel.swift

```diff
@@ -192,15 +192,18 @@ final class MLCardFormViewModel {
     }
     
     func groupIndexOfCardFormField(_ cardFormField: MLCardFormField) -> Int? {
-        let fieldId = cardFormField.property.fieldId()
-        let index = cardFormFields?.firstIndex(where: {
-            $0.first(where: {
-                $0.property.fieldId() == fieldId
-            }) != nil
-        })
-        
-        return index
-    }
+            let fieldId = cardFormField.property.fieldId()
+            guard let index = cardFormFields?.firstIndex(where: {
+                $0.first(where: {
+                    $0.property.fieldId() == fieldId
+                }) != nil
+            }) else { return nil }
+            
+            if fieldId == MLCardFormFields.expiration.rawValue {
+                return index
+            }
+            return Int(index + 1)
+        }
```

Comment:
I think has been shift to the right with an extra indentation.

Reply:
Ok i think i got it, thanks